### PR TITLE
Create new target

### PR DESCRIPTION
**Pull request by mato2593**
This PR includes the creation of a new target, with everything that it requires. There is also a little fix from the last demo

https://trello.com/c/ycuiOGvH/4-5-as-a-user-i-should-be-able-to-create-a-new-target

create-new-target

target-form-animation

### CODE REVIEW

File: Target-MVD-Mathias/ViewControllers/HomeViewController.swift [hunk 1]
```diff
@@ -85,18 +137,140 @@ extension HomeViewController: CLLocationManagerDelegate {
...
+    newTargetMarker.groundAnchor = CGPoint(x: 0.5, y: 0.5)
+    newTargetMarker.isFlat = true
+    newTargetMarker.map = mapView
+  }
+}
+
+extension HomeViewController: UITableViewDataSource {
+  
+  func numberOfSections(in tableView: UITableView) -> Int {
+    return 1
```
Comment (mjurfest) [inline Target-MVD-Mathias/ViewControllers/HomeViewController.swift]: This method is not needed. By default the number of sections is 1

Comment (mjurfest) [inline Target-MVD-Mathias/ViewControllers/HomeViewController.swift]: This method is not needed. By default the number of sections is 1

File: Target-MVD-Mathias/Views/TargetFormView.swift [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+      title = titleTextField.text ?? ""
+    }
+  }
+  
+}
+
+extension TargetFormView: UIPickerViewDelegate {
+  
+  func numberOfComponents(in pickerView: UIPickerView) -> Int {
+    return 1
```
Comment (mjurfest) [inline Target-MVD-Mathias/Views/TargetFormView.swift]: I am not sure but probably this default value will be 1, test it and if works without please remove this method, if not just leave it.

**Review by mjurfest — COMMENTED** [review]
Small comments

**Review by fernandatoledo — APPROVED** [review]

**Review by mjurfest — APPROVED** [review]

File: Target-MVD-Mathias/Networking/Services/TargetAPI.swift [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+//  Created by Mathias Cabano on 9/7/17.
+//  Copyright © 2017 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class TargetAPI {
+  
+  fileprivate static let topicsUrl = "/topics"
```
Comment (pMalvasio) [inline Target-MVD-Mathias/Networking/Services/TargetAPI.swift]: Add extra `/` at the end to keep consistency

**Review by pMalvasio — COMMENTED** [review]
Small comment

**Review by pMalvasio — APPROVED** [review]

**Pull request merged**